.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - checkout
       - run: luarocks make *.rockspec
       - run: /usr/local/openresty/luajit/bin/apicast-cli busted
-      - run: prove
+      - run: TEST_NGINX_BINARY=openresty-debug prove
```

Comment:
Should be fixed https://github.com/3scale/s2i-openresty/pull/26/commits/fc06b357496f77d4d5eaacb681b0370187d2e7b9